themes.txt

```diff
@@ -266,6 +266,7 @@ github.com/nirocfz/arabica
 github.com/nodejh/hugo-theme-cactus-plus
 github.com/NormandErwan/Blogpaper
 github.com/ntk148v/hugo-cuisine-book
+https://github.com/nunocoracao/blowfish
```

Comment:
You need to remove the protocol  -- see the other examples.

Reply:
Thanks for the help @bep, sorry I missed the comment... and also just copy-pasted without paying attention